Test case for StringSum

mnurzyns opened the pull request:
One test case is enought because StringSum class give us
only interface to sum small string with brackets betwixt.

Commit:
Test case for StringSum

One test case is enought because StringSum class give us
only interface to sum small string with brackets betwixt.

marcinwozniak0 commented on MessengerOnFileForLinux/MessageFramework/test/StringSumTest.cpp:
```diff
@@ -1,7 +1,14 @@
 {
-    EXPECT_TRUE(true);
+    StringSumSquareBrackets summedString;
+    std::string expectedString = "[This ][string ][is][ summed.]";
```
const

mnurzyns replied:
ok

marcinwozniak0 commented on MessengerOnFileForLinux/MessageFramework/test/StringSumTest.cpp:
```diff
@@ -1,7 +1,14 @@
 #include <StringSum.hpp>
 
-TEST(StringSumTest, trueIsTrue)
+TEST(StringSumTest, sumString)
```
Ta nazwa testu niewiele mówi. Może coś w stylu isComponentsArePlacedInSquareBracketsAndReturnAsContinuous string? Nie mam na razie lepszych pomysływó, może mi coś jeszcze przyjdzie do głowy albo tobie

mnurzyns replied:
Klasa nazywa się StringSum, jedyne co robi to cumuje stringi, czemu nie możemy nazwać testu, który sprawdza, że sumuje stringi sumString?

marcinwozniak0 replied:
Jak miałem jakiś czas temu szkolenie z Cpp WL, to gość mówił żeby nazwy testów dokumentowały dzialanie funkcjonalności. Czytasz nazwe testu, i wiesz bez patrzenia w implementacje tylko po samej nazwie co ona robi

mnurzyns replied:
No a ja czytałem Wujka Boba, że powinna mieć jak najmniej i mówić co jest testowane zamiast długich tasiemców, ale ok ta może jest zbyt krótka i mało precyzyjna:

isStringsAreSummedWithBrackets - co sądzisz o takiej?

marcinwozniak0 replied:
ok, akceptuje

Commit:
Change test name and add const to value

After review we decided that test should have better name.
Also value which is constatnt should be marked as const.
